README.md

```diff
@@ -1,16 +1,31 @@
 # Beginner Resources
 
-Beginner-resources is a repository designed to help beginners get started in the exciting world of web development. You will find here helpful tools and resources to get you started on your journey to becoming the best web developer you can be. Whether it's for front-end or back-end development, this repository will help.
+Beginner-resources is a repository designed to help beginners get started in the exciting world of development. You will find here helpful tools and resources to get you started on your journey to becoming the best developer you can be. Whether it's for front-end or back-end development, this collection of resources will help.
+
+Web Development Basics
```

Comment:
I think these headings would look nice with headings, like a `##` or `###`. GitHub will also add anchors to the headings that way, which means we can send people links directly to those parts of the pages.

What do you think about updating these three sections to use headings?